bootstrap.d/x11-libs.yml

```diff
@@ -503,8 +497,8 @@ packages:
     source:
       subdir: ports
       git: 'https://gitlab.freedesktop.org/xorg/lib/libx11.git'
-      tag: 'libX11-1.8.1'
-      version: '1.8.1'
+      tag: 'libX11-1.8.6'
```

Comment:
This port does not compile without a newer autoconf, I propose downgrading to `1.8.4`.

Reply:
I propose updating autoconf instead